api/src/controllers/planDeTransport/ligne-de-bus.js

```diff
@@ -127,7 +129,15 @@ router.get("/:id", passport.authenticate("referent", { session: false, failWithE
 
     const ligneToBus = await LigneToPointModel.find({ lineId: id });
 
-    return res.status(200).send({ ok: true, data: { ...ligneBus, ligneToBus } });
+    let meetingsPointsDetail = [];
+    for await (let line of ligneToBus) {
+      const pointDeRassemblement = await PointDeRassemblementModel.findById(line.meetingPointId);
+      meetingsPointsDetail.push({ ...line._doc, ...pointDeRassemblement._doc });
+    }
```

Comment:
A priori pour moi, t'as pas besoin de mettre un for await si tu mets un await à l'intérieur. Tu peux faire un for normal.